app/src/main/java/protect/card_locker/CsvDatabaseImporter.java

```diff
@@ -142,6 +142,10 @@ private void importLoyaltyCard(SQLiteDatabase database, DBHelper helper, CSVReco
             headerTextColor = extractInt(DBHelper.LoyaltyCardDbIds.HEADER_TEXT_COLOR, record, true);
         }
 
-        helper.insertLoyaltyCard(database, id, store, note, cardId, barcodeType, headerColor, headerTextColor);
+        //Starred status will not be imported
```

Comment:
Good point. In case you export/import the whole database it probably makes sense to import the starred status.
Just if you share only one card with someone else it probably makes no sense (this was my first thought). I can add the code for import of starred status.

Reply:
If you just share a single card using the build-in share feature, it doesn't share all fields anyway, just its own whitelist. So for CSV I would really just always import and save the starred status: https://github.com/brarcher/loyalty-card-locker/blob/master/app/src/main/java/protect/card_locker/ImportURIHelper.java

Reply:
I think it's now working correctly - CSV contains the starStatus and single card starStatus is not imported